internal/pubsub/drivertest/drivertest.go

```diff
@@ -184,7 +183,11 @@ func testSendReceive(t *testing.T, newHarness HarnessMaker) {
 	defer cleanup()
 
 	// Send to the topic.
-	var want []*pubsub.Message
+	type Message struct {
```

Comment:
Isn't this kind of fragile? We'll have to remember to update this if we ever add fields to `Message`.

Reply:
This type is only defined within this test. If we add fields to `Message` then this test will still work. If we add usage of those fields to this test then we will find out right away that we need to add them to this type.